### PR TITLE
Feature/best rated

### PR DESCRIPTION
**Pull request by dwoznicka**

### CODE REVIEW

File: client/js/components/pages/dashboard.jsx [hunk 1]
```diff
@@ -60,6 +60,16 @@ export default class Dashboard extends React.Component {
             </Card>
           </div>
         </div>
+        <div className='row' style={{marginBottom: '15px'}}>
+          <div className='col-xs-12 col-sm-12'>
```
Comment (vrael560) [inline client/js/components/pages/dashboard.jsx]: I think it by default take 100% width :P also any `col-xs-` class will apply to all resolutions if it's the only one provided

File: client/js/components/snippets/snippet.jsx [hunk 2]
```diff
@@ -61,11 +61,11 @@ export default class Snippet extends React.Component {
   }
 
   _checkRatingAbility() {
-    let today = Date.now();
-    let dateCreated = Date.parse(this.props.createdAt);
+    let today = new Date(Date.now());
```
Comment (vrael560) [inline client/js/components/snippets/snippet.jsx]: `new Date()` return current time without arguments

File: server/routes/dashboard.js [hunk 3]
```diff
@@ -37,7 +37,16 @@ router.get('/', authChecker, function(req, res) {
       mappedComments = comments.map(function (c){
         return c.toJson();
       });
-      res.status(200).send({ count: count, comments: mappedComments, snippets: mappedSnippets });
+    }).then(function () {
+      var today = new Date(Date.now());
```
Comment (vrael560) [inline server/routes/dashboard.js]: same as above

File: server/routes/dashboard.js [hunk 4]
```diff
@@ -37,7 +37,16 @@ router.get('/', authChecker, function(req, res) {
...
+      var firstDayOfMonth = new Date(today.getFullYear(), today.getMonth(), 1);
+      return models.Snippet.scope(['withVersions', 'withAuthor', { method: ['withRatings', req.user.get('id')] }])
```
Comment (vrael560) [inline server/routes/dashboard.js]: scope for current month snippet would be nice :)

File: client/js/components/pages/dashboard.jsx (hunk 1 above)
Comment (dwoznicka) [inline client/js/components/pages/dashboard.jsx]: it doesn't take 100% width by default ;) more of some random value as it's not 50% as well.. but i'll leave just `col-xs-12` and delete `col-sm`

**Pull request merged**